.travis.yml

```diff
@@ -5,7 +5,7 @@ services:
 env:
   matrix:
    - HHVM_VERSION=latest
-   - HHVM_VERSION=4.8-latest
```

Comment:
do you want to add `nightly` too?

Reply:
Nah, I've found that nightly sporadically breaks our build on language changes in ways that don't usually happen on latest. That's good in a sense of getting a heads up, but also spammy and puts a lot of X's in our commit history 😅